lib/src/components/NxTransferListHalf/NxTransferListHalf.tsx

```diff
@@ -149,6 +150,7 @@ export default function NxTransferListHalf<T extends string | number = string>(p
           </button>
         }
         {/* Add the tabIndex here to meet the a11y requirement that scrollable region must have keyboard access */}
+        <NxScrollRender>
```

Comment:
@rpokorny which tool should I use to apply format?  I used prettier but I found quite invasive